### PR TITLE
Parallel packer build and auto checksum calculation

### PR DESCRIPTION
**Pull request by Sarafian**
Regarding issue GH-66 and GH-67

### CODE REVIEW

File: Source/Invoke-PackerBuild.ps1 [hunk 1]
```diff
@@ -184,26 +190,28 @@ switch ($PSCmdlet.ParameterSetName) {
...
 try
 {
     $invokedPacker=$false
 
-    if ($PSCmdlet.ShouldProcess($packerFileNameName, "packer build")){
+    if ($PSCmdlet.ShouldProcess($packerFileName, "packer build")){
         $invokedPacker=$true
         $env:PACKER_LOG=1
-        $packetLogPath=Join-Path $env:TEMP "$($packerFileNameName).txt"
+        $packetLogPath=Join-Path $env:TEMP "$($packerFileName).txt"
```
Comment (beutepa) [inline Source/Invoke-PackerBuild.ps1]: The log path should include the ISHVersion.
Typo: packetLogPath -> packerLogpath

File: Source/Invoke-PackerBuild.ps1 [hunk 2]
```diff
@@ -6,8 +6,6 @@ param(
     [Parameter(Mandatory=$false,ParameterSetName="AWS EC2 AMI")]
     [Parameter(Mandatory=$true,ParameterSetName="Vagrant Hyper-V")]
     [string]$MockConnectionString=$null,
-    [Parameter(Mandatory=$false,ParameterSetName="AWS EC2 AMI")]
-    [string]$SourceAMI,
     [Parameter(Mandatory=$true,ParameterSetName="AWS EC2 AMI")]
     [string]$IAMInstanceProfile,
     [Parameter(Mandatory=$true,ParameterSetName="AWS EC2 AMI")]
```
Comment (beutepa) [inline Source/Invoke-PackerBuild.ps1]: The region should also be available for the ParameterSetName "Vagrant Hyper-V".
Maybe make this optional, but add a default value through New-ISHBootstrapAWSReferences.ps1 ?

File: Source/Invoke-PackerBuild.ps1 (hunk 1 above)
Comment (Sarafian) [inline Source/Invoke-PackerBuild.ps1]: Unfortunately ISHBootstrap's builders depends only on public artifacts and that means only ISH.12.0.3 at this moment

Comment (Sarafian) [inline Source/Invoke-PackerBuild.ps1]: Correct also `packet`

**Comment by Sarafian** [comment]
Close #66 
Closes #67

**Pull request merged**